From 372af57b902d95dfbc1b5f785010862e4b1eb396 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 16 May 2023 19:09:13 +0200
Subject: [PATCH] test: rename assert.sh to util.sh

So we can extend it with additional utility functions without making it
confusing.

No functional change.

(cherry picked from commit cb153b4fe9046590eb8efba66b67a0db899cb96d)

Related: RHEL-55301
---
 test/units/testsuite-17.02.sh     | 4 ++--
 test/units/testsuite-17.07.sh     | 4 ++--
 test/units/testsuite-17.08.sh     | 4 ++--
 test/units/testsuite-17.09.sh     | 4 ++--
 test/units/testsuite-17.12.sh     | 4 ++--
 test/units/testsuite-26.sh        | 4 ++--
 test/units/testsuite-35.sh        | 4 ++--
 test/units/testsuite-45.sh        | 4 ++--
 test/units/testsuite-58.sh        | 4 ++--
 test/units/testsuite-60.sh        | 4 ++--
 test/units/testsuite-65.sh        | 4 ++--
 test/units/testsuite-71.sh        | 4 ++--
 test/units/testsuite-73.sh        | 4 ++--
 test/units/testsuite-75.sh        | 4 ++--
 test/units/testsuite-76.sh        | 4 ++--
 test/units/{assert.sh => util.sh} | 2 +-
 16 files changed, 31 insertions(+), 31 deletions(-)
 rename test/units/{assert.sh => util.sh} (95%)

diff --git a/test/units/testsuite-17.02.sh b/test/units/testsuite-17.02.sh
index 82f9fd1f62..22b1552653 100755
--- a/test/units/testsuite-17.02.sh
+++ b/test/units/testsuite-17.02.sh
@@ -7,8 +7,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 mkdir -p /run/udev/rules.d/
 
diff --git a/test/units/testsuite-17.07.sh b/test/units/testsuite-17.07.sh
index 549107af10..50dc16b14b 100755
--- a/test/units/testsuite-17.07.sh
+++ b/test/units/testsuite-17.07.sh
@@ -3,8 +3,8 @@
 set -ex
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 wait_service_active() {(
     set +ex
diff --git a/test/units/testsuite-17.08.sh b/test/units/testsuite-17.08.sh
index f740b337f7..3502538fd4 100755
--- a/test/units/testsuite-17.08.sh
+++ b/test/units/testsuite-17.08.sh
@@ -3,8 +3,8 @@
 set -ex
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 # This is a test for issue #24518.
 
diff --git a/test/units/testsuite-17.09.sh b/test/units/testsuite-17.09.sh
index 01ac5f1709..c3914d9356 100755
--- a/test/units/testsuite-17.09.sh
+++ b/test/units/testsuite-17.09.sh
@@ -3,8 +3,8 @@
 set -ex
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 # This is a test for issue #24987.
 
diff --git a/test/units/testsuite-17.12.sh b/test/units/testsuite-17.12.sh
index df74d356ee..ccc91bfe46 100755
--- a/test/units/testsuite-17.12.sh
+++ b/test/units/testsuite-17.12.sh
@@ -3,8 +3,8 @@
 set -ex
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 create_link_file() {
     name=${1?}
diff --git a/test/units/testsuite-26.sh b/test/units/testsuite-26.sh
index debee91dde..340e02a561 100755
--- a/test/units/testsuite-26.sh
+++ b/test/units/testsuite-26.sh
@@ -3,8 +3,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 : >/failed
 
diff --git a/test/units/testsuite-35.sh b/test/units/testsuite-35.sh
index 1863c535a4..18c50279e2 100755
--- a/test/units/testsuite-35.sh
+++ b/test/units/testsuite-35.sh
@@ -3,8 +3,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 cleanup_test_user() (
     set +ex
diff --git a/test/units/testsuite-45.sh b/test/units/testsuite-45.sh
index 7e757e4b00..c85e17da7b 100755
--- a/test/units/testsuite-45.sh
+++ b/test/units/testsuite-45.sh
@@ -4,8 +4,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 restore_timezone() {
     if [[ -f /tmp/timezone.bak ]]; then
diff --git a/test/units/testsuite-58.sh b/test/units/testsuite-58.sh
index f41069ee04..850111b319 100755
--- a/test/units/testsuite-58.sh
+++ b/test/units/testsuite-58.sh
@@ -8,8 +8,8 @@ if ! command -v systemd-repart &>/dev/null; then
     exit 0
 fi
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 export SYSTEMD_LOG_LEVEL=debug
 export PAGER=cat
diff --git a/test/units/testsuite-60.sh b/test/units/testsuite-60.sh
index a29364568d..920f5019d2 100755
--- a/test/units/testsuite-60.sh
+++ b/test/units/testsuite-60.sh
@@ -3,8 +3,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 teardown_test_dependencies() (
     set +eux
diff --git a/test/units/testsuite-65.sh b/test/units/testsuite-65.sh
index 0fb143bde7..4e12511a1f 100755
--- a/test/units/testsuite-65.sh
+++ b/test/units/testsuite-65.sh
@@ -3,8 +3,8 @@
 # shellcheck disable=SC2016
 set -eux
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 # On RHEL9 we don't have the `util.sh` script, so we need to define the `runas` function here
 runas() {
diff --git a/test/units/testsuite-71.sh b/test/units/testsuite-71.sh
index 2382cccb3a..49efa4f982 100755
--- a/test/units/testsuite-71.sh
+++ b/test/units/testsuite-71.sh
@@ -4,8 +4,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 restore_hostname() {
     if [[ -e /tmp/hostname.bak ]]; then
diff --git a/test/units/testsuite-73.sh b/test/units/testsuite-73.sh
index f9e2dce1bf..279507af31 100755
--- a/test/units/testsuite-73.sh
+++ b/test/units/testsuite-73.sh
@@ -4,8 +4,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 enable_debug() {
     mkdir -p /run/systemd/system/systemd-localed.service.d
diff --git a/test/units/testsuite-75.sh b/test/units/testsuite-75.sh
index ddd86d09bb..370a91863f 100755
--- a/test/units/testsuite-75.sh
+++ b/test/units/testsuite-75.sh
@@ -11,8 +11,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 : >/failed
 
diff --git a/test/units/testsuite-76.sh b/test/units/testsuite-76.sh
index cb571f8b97..0436afec65 100755
--- a/test/units/testsuite-76.sh
+++ b/test/units/testsuite-76.sh
@@ -3,8 +3,8 @@
 set -eux
 set -o pipefail
 
-# shellcheck source=test/units/assert.sh
-. "$(dirname "$0")"/assert.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
 
 export SYSTEMD_LOG_LEVEL=debug
 
diff --git a/test/units/assert.sh b/test/units/util.sh
similarity index 95%
rename from test/units/assert.sh
rename to test/units/util.sh
index 2f4d93ab8c..00b8c5e393 100755
--- a/test/units/assert.sh
+++ b/test/units/util.sh
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
-# utility functions for shell tests
+# Utility functions for shell tests
 
 assert_true() {(
     set +ex
